src/server/main_service.cc

```diff
@@ -1448,7 +1453,7 @@ void Service::RegisterCommands() {
       << CI{"PSUBSCRIBE", CO::NOSCRIPT | CO::LOADING, -2, 0, 0, 0}.MFUNC(PSubscribe)
       << CI{"PUNSUBSCRIBE", CO::NOSCRIPT | CO::LOADING, -1, 0, 0, 0}.MFUNC(PUnsubscribe)
       << CI{"FUNCTION", CO::NOSCRIPT, 2, 0, 0, 0}.MFUNC(Function)
-      << CI{"MONITOR", CO::ADMIN, 1, 0, 0, 0}.MFUNC(Monitor)
+      << CI{"MONITOR", CO::ADMIN | CO::NO_AUTOJOURNAL, 1, 0, 0, 0}.MFUNC(Monitor)
```

Comment:
Why do you set CO::NO_AUTOJOURNAL? this is not write command, will not be sent to replica